src/NetSparkle/AppCastHandlers/XMLAppCast.cs

```diff
@@ -30,24 +35,40 @@ public class XMLAppCast : IAppCastHandler
         /// <summary>
         /// App cast title (usually the name of the application)
         /// </summary>
-        public string Title { get; set; }
+        public string Title
+        {
+            get => _title;
+            set => _title = value;
+        }
 
         /// <summary>
         /// App cast language (e.g. "en")
         /// </summary>
-        public string Language { get; set; }
+        public string Language
+        {
+            get => _language;
+            set => _language = value;
+        }
 
         /// <summary>
         /// Extension (WITHOUT the "." at the start) for the signature
         /// file. Defaults to "signature".
         /// </summary>
-        public string SignatureFileExtension { get; set; }
+        public string SignatureFileExtension
+        {
+            get => _signatureFileExtension;
+            set => _signatureFileExtension = value;
+        }
 
         /// <summary>
         /// List of <seealso cref="AppCastItem"/> that were parsed in the app cast
         /// </summary>
         public readonly List<AppCastItem> Items;
 
+        private string _title;
+        private string _language;
```

Comment:
What's the benefit to using the private vars here if you're just going to have plain getters and setters?

Reply:
None, must have been a Rider/refactoring thing - now it's back to plain getter/setter.